README.md

```diff
@@ -230,6 +230,10 @@ __Parameters__:
 
 - Download requests is being cached by native implementation. To avoid caching, pass `if-Modified-Since` header to download method.
 
+### Windows Quirks
+
+- Usage of [BackgroundDownloader](https://msdn.microsoft.com/en-us/library/windows/apps/windows.networking.backgroundtransfer.backgrounddownloader.aspx) entails the latency issues on Windows Phone. You can use XHR or [HttpClient](https://msdn.microsoft.com/en-us/library/windows/apps/windows.web.http.httpclient.aspx) as a quicker alternative for small downloads.
```

Comment:
Just "Windows", not "Windows Phone". I'm pretty sure this is not a phone-only issue since the APIs are the same on Windows desktop.

Also, is the reader going to understand what the "latency issues" are, or does it need more explanation here?